runtime/dancebox/src/lib.rs

```diff
@@ -613,14 +609,10 @@ construct_runtime!(
         AuthorityAssignment: pallet_authority_assignment = 25,
 
         // Collator support. The order of these 4 are important and shall not change.
-        Authorship: pallet_authorship = 30,
-        CollatorSelection: pallet_collator_selection = 31,
-        Session: pallet_session = 32,
-        Aura: pallet_aura = 33,
-        AuraExt: cumulus_pallet_aura_ext = 34,
-        AuthorityMapping: pallet_authority_mapping = 35,
-
-        AuthorInherent: pallet_author_inherent = 50,
+        CollatorSelection: pallet_collator_selection = 30,
+        Session: pallet_session = 31,
```

Comment:
Does this change the well known key? (probably not because it is name based right?)

Reply:
Keys are name based yeah, so indices are not important. We should probably cleanup the indices before a release.

Reply:
Yeah I am not too worried about the indices for now then. I think they are mostly clean the way they are, but if we want to change them, it will only affect in cases where other chains depend on this (which mostly I think its extrinsic-based, i.e., XCM only). So for now we should be on the safe side doing a release